src/en/charms-storage.md

```diff
@@ -237,6 +238,53 @@ default values:
 juju deploy postgresql --storage pgdata=ebs,10G cache=ebs-ssd
 ```
 
+### Remove storage
+
+If you remove a unit that has storage, the storage is removed with the unit.
+
+The long term plan is to enable storage to be managed separately from units
+for types of storage that can be persistent, such as EBS volumes. This will
+enable the storage to be reused by new units. We have not fully implemented
+this ability. One of the first steps in managing the storage life cycle
+separately is allowing you to remove existing storage, which you can do today.
+
+When you remove the storage attached to a running charm, `remove-storage` will
+run the storage-detaching hook, detach the storage from the machine, and remove
+the storage from the model, destroying the cloud storage resources in the
+process.
+
+To remove existing storage, you must specify one or more unit storage, volume, or
+filesystem IDs; these can be found in the output of `juju storage`, shown here:
+
+```bash
+[Storage]
+Unit          Id        Location     Status    Message
+postgresql/0  pgdata/0  /srv/pgdata  attached 
+```
+
+To remove this storage by specifying the unit storage instance, use:
+
+```bash
+juju remove-storage pgdata/0
+```
+
+This will also remove the associated volume and filesystem. You can reference
+just the volume or the filesystem using the IDs when removing storage. To
+eliminate confusion over which is being referred to, you must add either
+`volume-` or `filesystem-` to the IDs, as shown here:
+
+```bash
+juju remove-storage volume-0
+juju remove-storage filesystem-0
+```
+
+If you have and want to remove the storage from multiple volumes or filesystems
+at the same time, you can refer to more than one in the command, like this:
+
+```bash
+juju remove-storage filesystem-2/3
```

Comment:
Okay. We will keep this as it is, unmerged, until we hear from you, @axw. Thanks!